### PR TITLE
Added `get:release-line` command

### PR DESCRIPTION
**Pull request by fritzmonkey**
## Summary
Added `get:release-line` command

### CODE REVIEW

File: src/commands/get/release-line.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const { wrhsHost, auth } = this.mergeConfig(flags);
+
+    if (!wrhsHost) {
+      return this.error(this.missingHostError());
+    }
+
+    const wrhs = this.wrhs(auth, { wrhsHost });
+
+    return new Promise((resolve, reject) => {
+      wrhs.releaseLine.get({ pkg, version }, (err, releaseLine) => {
```
Comment (msluther) [inline src/commands/get/release-line.js]: I assume this is okay with an undefined version?

File: src/commands/get/release-line.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+ReleaseLineCommand.description = `Gets information about a given release line from warehouse.
+`;
+
+ReleaseLineCommand.args = [{
+  name: 'package',
+  required: true,
+  description: 'The package and version (optional) to get the release line for'
+}];
```
Comment (msluther) [inline src/commands/get/release-line.js]: Should version be listed as an arg here? Or is version of the form`pkg@1.2.3`? If the latter, you should doc that in the description here.

Comment (msluther) [reply src/commands/get/release-line.js]: Read the tests after. It is `pkg@1.2.3`. Can we update the description to say `package@version`?

File: src/commands/get/release-line.js (hunk 1 above)
Comment (jcrugzz) [reply src/commands/get/release-line.js]: Yea the client handles it

**Pull request merged**